lib/grammers-client/src/client/messages.rs

```diff
@@ -751,6 +751,35 @@ impl Client {
         Ok(message_ids.iter().map(|id| map.remove(id)).collect())
     }
 
+    /// Get a message using your ID.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # async fn f(chat: grammers_client::types::Chat, mut client: grammers_client::Client) -> Result<(), Box<dyn std::error::Error>> {
+    /// let message_id = 123;
+    ///
+    /// let message = client.get_message_by_id(&chat, message_id).await?;
```

Comment:
It's unfortunate that the API *technically* lets you fetch a message by ID without using the chat, but I guess it's fine to hide this from the user should that ever change.

Reply:
I agree